### ISSUE TITLE
AIR SDK for Linux Mouse Right click on textFiled crash the application

### ISSUE THREAD

**Issue by saggelopoulos** [airsdk/Adobe-Runtime-Support#1255]
## Problem Description

Describe your problem in detail. Include the following information:

* AIR SDK 33.1.1.633? 
* Ubuntu Linux 20.04 64bit.

## Steps to Reproduce
Compile and package the code  under a Linux . Run the application and Position the cursor in a text field and press the  mouse right button. The application crash with:  "Application crashed with an unhanded SIGSEGV" message  
 

<?xml version="1.0" encoding="utf-8"?>
<s:WindowedApplication xmlns:fx="http://ns.adobe.com/mxml/2009" 
					   xmlns:s="library://ns.adobe.com/flex/spark" 
					   xmlns:mx="library://ns.adobe.com/flex/mx">
	<mx:TextInput x="10" y="32" width="120" id="test" text="" toolTip="enter the company name "/>
	<s:TextInput x="10" y="62" width="120" id="test1" text="" toolTip="enter the company name "/>
	<mx:Button x="10" y="150" width="100" id="button1" label="button1" />
	
</s:WindowedApplication>

**Comment by ajwfrost** [comment]
Thanks - I just checked with one of the engineers who was looking at the linux issues, this one is related to a thread clash between X11 and GDK, we have a fix for it which just needs a little more testing. This is the same root cause as #1253

**Comment by ajwfrost** [comment]
Yes, sorry, this will be fixed in our next release, we're aiming for the first week of November for this...

**Comment by pol2095** [comment]
The app not crash on Ubuntu now, but copy, paste is not visible in the menu ?
